app/src/test/java/nl/tudelft/b_b_w/model/DatabaseHandlerUnitTest.java

```diff
@@ -42,7 +42,7 @@
     private final String iban = "iban";
     private final String publicKey = "publicKey";
     private Block _block;
-    private final int trustValue = 0;
+    private final int trustValue = TrustValues.INITIALIZED.getValue();
```

Comment:
Could you give me a link and I prefer not a link to a stackoverflow discussion, where multiple people have different reasonings, which are all correct? Next to that, the value itself does not change in the scope, so the when you use a shorter variable name it is more readable.